src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WSALT

```diff
@@ -1,5 +1,5 @@
 {"name" : "WSALT", "sections" : ["SCHEDULE"],
-"comment" : "The CONCENTRATION item should be UDA - but UDA & size_type:ALL is not supported",
+"comment" : "UDA & size_type:ALL is not supported i.e Multi-Component Brine Model not supported",
 "items" :
       [{"name" : "WELL"         , "value_type" : "STRING"},
-       {"name" : "CONCENTRATION" , "value_type" : "DOUBLE" , "size_type" : "ALL"}]}
```

Comment:
To support an arbiotrary number of columns - this must be `size_type: ALL`

Reply:
There is an issue with conversion and UDQ when "ALL"  is used. There is no plan for supporting transport of ions so I think we can live without "ALL" here until the UDQ issue is resolved.